### PR TITLE
add clarity in contributing.md on application of labels

### PR DESCRIPTION
**Pull request by jsms90**
Added explanation on what I assume are non-contentious:
+ week labels
+ bug
+ discuss
+ question
+ help wanted

More to come before #316 can be closed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,11 +9,20 @@ The following people are available to review pull requests: [@bradreeder](https:
...
  6. Once you are sure of what you need to do and that it is needed, assign yourself to the issue.
  7. Clone, and create a new branch for your work
 
+### Labels
+
+`week-<name-here>` - if your new issue relates to a specific week of the course (e.g. `week-toolkit`)
+`week-epic` - you are conducting a planning phase which encompass many changes to a particular week
+`bug` - something in this repo is broken (e.g. a link)
+`discuss` - you need the community's decision before you are able to complete any work
+`question` - you're not actually sure whether this is any issue or not and would like confirmation
```
Comment (eliascodes) [inline CONTRIBUTING.md]: Typo: "any" -> "an"

Just for the record, I've typically understood `question` issues to be those asking for clarification or open questions, that can be addressed without changes in source.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,11 +9,20 @@ The following people are available to review pull requests: [@bradreeder](https:
...
  5. Indicate on the issue that you would like to take it on. Assign one of the reviewers above if you need a response.
  6. Once you are sure of what you need to do and that it is needed, assign yourself to the issue.
  7. Clone, and create a new branch for your work
 
+### Labels
+
+`week-<name-here>` - if your new issue relates to a specific week of the course (e.g. `week-toolkit`)
+`week-epic` - you are conducting a planning phase which encompass many changes to a particular week
+`bug` - something in this repo is broken (e.g. a link)
+`discuss` - you need the community's decision before you are able to complete any work
```
Comment (eliascodes) [inline CONTRIBUTING.md]: Is "decision" here right? Shouldn't it be something like "you'd like the community's input before you start any work"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,11 +9,20 @@ The following people are available to review pull requests: [@bradreeder](https:
...
+`question` - you're not actually sure whether this is any issue or not and would like confirmation
+`help wanted` - this is a big task and you would like some help to complete it
```
Comment (eliascodes) [inline CONTRIBUTING.md]: Doesn't need to be a big task?

**Comment by jsms90** [comment]
:heart: Feel free to merge @nelsonic

**Pull request merged**